sources/net/Network2.cpp

```diff
@@ -380,13 +380,13 @@ void SocketHandler::sendMessage(Message2 message)
 
     QByteArray data;
 
-    uchar temp[4];
+    char temp[4];
```

Comment:
Yes I noticed that, refactored it as a reinterpret_cast instead since it is more in line with C++ style casting.